### PR TITLE
V.2.3 authenticated users and RN 0.60.+

### PR DESCRIPTION
**Pull request by stephenkiers**
This PR includes `setUserId`, `clearAllState`, RN 0.60.+ support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,14 @@ Manage pusher interest subscriptions from within React Native JS
 **UPDATE: 22/09/2019**
 I appreciate all of the support on this project, however due to changes in work I haven't done much with RN in the last year.  Because I know this package helps a lot of people out, I'm looking for and happy to include some else willing to maintain this package, testing and approving PRs.  Shoot me an email.
 
-**IMPORTANT!!!**
-
-This module is intended to complement the setup with [Carthage](https://pusher.com/docs/beams/getting-started/ios/sdk-integration#install-from-carthage). This module allows that implementation to be accessed directly from React Native JS.
-
-```Cartfile
-github "pusher/push-notifications-swift" ~> 2.0.0
-```
```
Comment (stephenkiers) [inline README.md]: no longer suggesting carthage

File: README.md [hunk 2]
```diff
@@ -5,20 +5,14 @@ Manage pusher interest subscriptions from within React Native JS
...
 
 More information about Pusher Beams and their Swift library, `push-notifications-swift`, can be found on their [Github repo](https://github.com/pusher/push-notifications-swift).
 
-There are some issues installing this repo you need to be aware of. More info can be found in [issue 65](https://github.com/pusher/push-notifications-swift/issues/65).
```
Comment (stephenkiers) [inline README.md]: there are out of date

File: README.md [hunk 3]
```diff
@@ -5,20 +5,14 @@ Manage pusher interest subscriptions from within React Native JS
...
-
 [![npm version](https://badge.fury.io/js/react-native-pusher-push-notifications.svg)](https://badge.fury.io/js/react-native-pusher-push-notifications)
 
+## Requirements
+**This branch is only compatible with React Native >0.60.x**
```
Comment (stephenkiers) [inline README.md]: _I think_

**Comment by stephenkiers** [comment]
Can we get this merged?

**Comment by mikebridge** [comment]
@b8ne I would merge this but it doesn't look like I have administrative permission any more.

**Comment by nagad814** [comment]
@b8ne please can you merge this branch. I would like to use this in a project.

**Comment by ewal** [comment]
Longing for this to be merged too. It's currently the only library that preventing us from upgrading to RN 0.60 :)

**Comment by emilkarl** [comment]
Oooh lets push it!

**Comment by livfwd** [comment]
@b8ne could you weigh-in on the best way for us to continue to push work forward here? Thanks!

**Pull request merged**